ts/client/src/debug-scripts/debug-perp.ts

```diff
@@ -0,0 +1,142 @@
+import { AnchorProvider, Wallet } from '@project-serum/anchor';
+import { Cluster, Connection, Keypair } from '@solana/web3.js';
+import * as dotenv from 'dotenv';
+import fs from 'fs';
+import { MangoClient } from '../client';
+import { MANGO_V4_ID } from '../constants';
+dotenv.config();
+
+const CLUSTER_URL =
+  process.env.CLUSTER_URL_OVERRIDE || process.env.MB_CLUSTER_URL;
+const PAYER_KEYPAIR =
+  process.env.PAYER_KEYPAIR_OVERRIDE || process.env.MB_PAYER_KEYPAIR;
```

Comment:
Why does this script need the admin keypair? I think we should pass in the group pubkey.